### PR TITLE
New integration: comfoair

### PR DESCRIPTION
**Pull request by mtdcr**
**Description:**
This documents the new comfoair integration. It's based on comfoconnect.markdown.

**Pull request in home-assistant:** home-assistant/home-assistant#27961

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/comfoair.markdown [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+---
+
+The `comfoair` integration lets you control Zehnder ComfoAir [350](https://www.international.zehnder-systems.com/products-and-systems/comfosystems/zehnder-comfoair-350) ventilation units from Home Assistant. A serial connection is needed to connect the unit to Home Assistant.
+
+The integration has a fan platform to view and control the ventilation speed, and a sensors platform to read out various temperatures. It passively listens to messages exchanged between CC Ease control units and the system board. It can take control of the system temporarily to set fan speeds.
+
+To set it up, add the following information to your `configuration.yaml` file:
+
+```yaml
+comfoair:
```
Comment (klaasnicolaas) [inline source/_integrations/comfoair.markdown:20]: We would like to see only required config variables in the first example

**Review by klaasnicolaas — DISMISSED** [review]

File: source/_integrations/comfoair.markdown (hunk 1 above)
Comment (mtdcr) [reply source/_integrations/comfoair.markdown:20]: Fixed.

**Comment by mtdcr** [comment]
@klaasnicolaas, the requested changes were addressed, but the status doesn't indicate it. It this something you should change manually?

**Review by frenck — CHANGES_REQUESTED** [review]
This integration exposes a service, which is not documented. Please be sure to add it.

**Comment by mtdcr** [comment]
> This integration exposes a service, which is not documented. Please be sure to add it.

Oops! Done. Thanks for your review!

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` needs an update before merging this one in.

**Pull request closed without merging**